# This service requests ...

string sensor
---
sensor_msgs/PointCloud2 simulated_scan